lib/web/websocket/frame.js

```diff
@@ -89,6 +89,49 @@ class WebsocketFrameSend {
 
     return buffer
   }
+
+  /**
+   * @param {string | Uint8Array} body
+   */
+  static createFastTextFrame (body) {
+    const maskKey = generateMask()
+
+    const buffer = typeof body === 'string' ? Buffer.from(body) : body
```

Comment:
when can body not be a string?

Reply:
It made this change for #3394, which provides a buffer instead of a string.

Reply:
I addressed it in #3394